invenio_app_ils/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -19,6 +19,23 @@
         "items": {
           "type": "keyword"
         },
+        "keywords": {
+          "type": "object",
+          "properties": {
+            "$schema": {
```

Comment:
I would not add the $schema for resolved references.
And shouldn't it be a list and not an object?

Reply:
I don't think ES mappings have an `array` type since types by default can have 0 or more values. We can use the `nested` datatype if we need to be able to query individual keywords in the keywords list. Is that something we might need?

I agree about $schema, no need for it in the resolved references.

Reply:
Oh right! I read now: https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html
Shall we check together before taking a decision? We just need to exactly understand how this will affect search.